Scoop/Controller/TripExtraDetailsViewController.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  TripExtraDetailsViewController.swift
+//  Scoop
+//
+//  Created by Elvis Marcelo on 3/18/22.
+//
+
+import UIKit
+
+class TripExtraDetailsViewController: UIViewController {
+    
+    private let travelersLabel = UILabel()
+    private let toLabel = UILabel()
+    private let count1Field = UITextField()
+    private let count2Field = UITextField()
+    
+    private let dateLabel = UILabel()
+    private let dateField = UITextField()
+    
+    private let timeLabel = UILabel()
+    private let timeField = UITextField()
+    
+    private let detailsLabel = UILabel()
+    private let detailsField = UITextField()
+
+
+    override func viewDidLoad() {
+        
+
+        
+        self.navigationItem.title = "Trip Details"
+        view.backgroundColor = .white
+        
+        travelersLabel.text = "Number of Travelers"
+        travelersLabel.textColor = .black
+        view.addSubview(travelersLabel)
+        
+        let peopleImage = UIImage(systemName: "person.2")
+        let peopleView = UIImageView(image: peopleImage)
+        peopleView.tintColor = .black
+        view.addSubview(peopleView)
+        
+        count1Field.placeholder = "0"
+        count1Field.textColor = .black
+        view.addSubview(count1Field)
+        
+        toLabel.text = "to"
+        toLabel.textColor = .black
+        view.addSubview(toLabel)
+        
+        count2Field.placeholder = "0"
+        count2Field.tintColor = .black
+        view.addSubview(count2Field)
+        
+        
+        dateLabel.text = "Date of Trip"
+        dateLabel.textColor = .black
+        view.addSubview(dateLabel)
+        
+        let calendarView = UIImageView(image: UIImage(systemName: "calendar"))
+        calendarView.tintColor = .black
+        view.addSubview(calendarView)
+        
+        dateField.placeholder = "mm/dd/yyyy"
+        dateField.textColor = .black
+        view.addSubview(dateField)
+        
+        
+        timeLabel.text = "Time of Trip"
+        timeLabel.textColor = .black
+        view.addSubview(timeLabel)
+        
+        let clockView = UIImageView(image: UIImage(systemName: "clock"))
+        clockView.tintColor = .black
+        view.addSubview(clockView)
+        
+        timeField.placeholder = "1:33 AM"
+        timeField.textColor = .black
+        view.addSubview(timeField)
+        
+        
+        detailsLabel.text = "Other Details"
+        detailsLabel.textColor = .black
+        view.addSubview(detailsLabel)
+        
+        let writeView = UIImageView(image: UIImage(systemName: "square.and.pencil"))
+        writeView.tintColor = .black
+        view.addSubview(writeView)
+        
+        detailsField.placeholder = "enter details..."
```

Comment:
I don't have access to the designs but should this be capitalized (since all the previously placeholders were).